simtools/applications/calculate_trigger_rate.py

```diff
@@ -97,11 +97,63 @@ def _parse(label, description):
         action="store_true",
     )
 
+    config.parser.add_argument(
+        "--energy_range",
+        help="Energy range (TeV) passed as a list of floats, the minimum and maximum, "
+        "respectively.",
+        required=False,
+        nargs=2,
+        type=str,
```

Comment:
Type = 'str', and help requests floats? Not clear if this is consistent.

Reply:
This is a good point. For some reason the configurator did not manage to add a parameter, which is a list of two floats, so I decided to get a list of str and transform to float in the code. If you happen to know the reason why type=float does not work here, please me know.

Reply:
Don't know. This probably requires to look a bit deeper into argparse.

Reply:
Yes, definitely not for this PR. I think it is fine as it is but if we really want to understand the reason for it we would have to look into argparse. I opened an issue for that #1005.